api/src/main/java/org/openmrs/Concept.java

```diff
@@ -70,57 +70,57 @@
  */
 @FullTextFilterDefs( { @FullTextFilterDef(name = "termsFilterFactory", impl = TermsFilterFactory.class) })
 public class Concept extends BaseOpenmrsObject implements Auditable, Retireable, Serializable, Attributable<Concept>,Customizable<ConceptAttribute> {
-	
+
 	public static final long serialVersionUID = 57332L;
-	
-	private static final Logger log = LoggerFactory.getLogger(Concept.class);
```

Comment:
Hii @Devanshk9, is your IDE properly configured to follow OpenMRS Code Style? I see unwanted indentation changes, wait for mentors to have a say on this.

Reply:
@PrathameshMutkure check now